### PR TITLE
Typing Speed test file added

### LINKED ISSUE #720: Typing speed test
I would like to add a python script on typing speed test. 
### Details:
We have to input sentences and it would show the time taken for typing, wpm and accuracy.

### PR DESCRIPTION
**Pull request by dhriti987**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [ ] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Typing-Speed-Test/speed_test.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import tkinter as tk
+import random
+import timeit
+
+start_time = 0
+
+def get_sentence():
+    global sentence,sentence_length,sentence_words
+    Reset()
+    with open("sentences.txt","r") as f:
```
Comment (kaustubhgupta) [inline Typing-Speed-Test/speed_test.py:10]:
```suggestion
    with open("./Typing-Speed-Test/sentences.txt","r") as f:
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by dhriti987** [comment]
@kaustubhgupta  Done

**Review by kaustubhgupta — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]
Nice @dhriti987 💯

**Comment by santushtisharma10** [comment]
@antrikshmisri

**Comment by santushtisharma10** [comment]
@Kushal997-das

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM

**Pull request merged**